packages/inter-protocol/src/vaultFactory/vaultDirector.js

```diff
@@ -437,6 +447,12 @@ const prepareVaultDirector = (
             allManagersDo(vm => vm.lockOraclePrices());
           });
         },
+        async updateShortfallReporter(newInvitation) {
```

Comment:
`updateShortfallReporter` has another meaning in this same module. please rename. consider `setShortfallReporter`.